src/PerfView/MainWindow.xaml.cs

```diff
@@ -2055,7 +2055,7 @@ private void Window_Closed(object sender, EventArgs e)
                 DoAbort(null, null);
             }
 
-            Environment.Exit(0);        // TODO can we do this another way?
+            // Environment.Exit(0);        // TODO can we do this another way?
```

Comment:
I can see the process exits when tests run, and I could not figure out when it is needed to close the application explicitly.